.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "test/3rdParty/babylon"]
+	path = test/3rdParty/babylon
+	url = https://github.com/babel/babylon.git
```

Comment:
How about to change [this](https://github.com/cst/cst/blob/a90cebc7d88304d55968d06d190fdfc6f658fe52/package.json#L51) to `babel/babylon#6.13.1`?

Still sucks though, since we not get all versions under caret

babel/babylon#6.9.1